### DISCUSSION TITLE
directory of packages / bundles?

### DISCUSSION THREAD

**Discussion by wnunezc** [symfony/symfony#44423] [Symfony Book: The Fast Track] [answered]
Is there a directory of packages / bundles created for symfony where you can see the version compatibility and a brief description of the features they include?

**Comment by stof** [comment, marked as answer]
https://packagist.org/?type=symfony-bundle gives you all the packages registered on Packagist with the `symfony-bundle` type, which should find lots of them (though not guaranteed to be exhaustive, as there is no requirement to use that type).
For compatibility, Packagist shows you the version constraints for each version of a package when you look at it.
